vision_params:
  camera:
    index: 2
    flip: true
  detection:
    margin: 70
    nwindows: 50
    minpix: 300
    track_width_px: 600
  thresholds: 
    binary_threshold: [160,255]
    sobel_threshold: [20,100]
  perspective:
    src_pc: [[0.03, 0.25], [0.97, 0.25], [1, 1], [0, 1.0]]
  node_config:
    rate: 20 #Frequency in Hz
    debug_mode: true